chmielubot/emojify.py

```diff
@@ -0,0 +1,57 @@
+from discord.ext import commands
+import discord
+import re
+
+class Emojify(commands.Cog):
```

Comment:
Emojify jest zbyt małą komendą, aby tworzyć dla niej cały nowy moduł. Wybierzmy bardziej generalną nazwę.
```suggestion
class Utility(commands.Cog):
```

Reply:
Przydałoby się jeszcze zmienić nazwę pliku i __init__.py